docs/sphinx/sections/api/apidocs/libraries/dagster-gcp.rst

```diff
@@ -41,6 +41,12 @@ BigQuery Ops
 
 .. autofunction:: import_gcs_paths_to_bq
 
+
+Data Quality
```

Comment:
We should call this Data Freshness. In our messaging, we distinguish between the two.

Reply:
ack.